sentry-ruby/lib/sentry/transaction.rb

```diff
@@ -51,12 +46,7 @@ def to_hash
 
     def start_child(**options)
```

Comment:
Looking at [Python](https://github.com/getsentry/sentry-python/blob/19fa43fec5a20b3561a16970ce395c93ac1be57d/sentry_sdk/tracing.py#L218-L241) and [JS](https://github.com/getsentry/sentry-javascript/blob/438ed91bd457f897da89c7728fd074bbef0a0ce1/packages/tracing/src/span.ts#L167-L185), there is probably no reason to have `Transaction.start_child` different than `Span.start_child` -- inheritance should be enough.